### PR TITLE
fix(binding-http): fix content negotiation

### PR DESCRIPTION
**Pull request by JKRhb**
This PR should resolve https://github.com/eclipse/thingweb.node-wot/issues/905. I already tested the examples in the browser and they are working again :) I will also another test case so that this is also asserted by the CI.

### CODE REVIEW

**Review by relu91 — APPROVED** [review]

File: packages/binding-http/src/http-server.ts [hunk 1]
```diff
@@ -544,16 +544,27 @@ export default class HttpServer implements ProtocolServer {
 
     private async handleTdRequest(thing: ExposedThing, req: http.IncomingMessage, res: http.ServerResponse) {
         const td = thing.getThingDescription();
-        let accept = req.headers.accept;
         const contentSerdes = ContentSerdes.get();
 
+        // TODO: Parameters need to considered here as well
```
Comment (danielpeintner) [inline packages/binding-http/src/http-server.ts:549]: Nit: need to **be** considered

**Review by danielpeintner — APPROVED** [review]
LGTM

One minor issue in a comment, which we could fix before merging...

File: packages/binding-http/src/http-server.ts (hunk 1 above)
Comment (JKRhb) [reply packages/binding-http/src/http-server.ts]: Thank you :) Should be fixed now.

Comment (danielpeintner) [reply packages/binding-http/src/http-server.ts]: 👍

**Pull request merged**